keyboard/inline.ts

```diff
@@ -5,15 +5,32 @@ type InternalButton = {
   text: string
 }
 
-export default function inlineKFunction(num: number, ...buttons: InternalButton[]) {
+export default function inlineKFunction(num: number, page: number, ...buttons: InternalButton[]) {
```

Comment:
```suggestion
export default function inlineKFunction(num: number, buttons: InternalButton[], page = 1) {
```
qogan joylada ham shunchaki array berish kerak, pageni olib tashlab

Reply:
agree, but I can't suggest it like this now, I've to change it from other places that called this function